src/hooks/v1/contractReader/OwnerOfProject.ts

```diff
@@ -4,7 +4,7 @@ import { V1ContractName } from 'models/v1/contracts'
 import useContractReader from './ContractReader'
 
 /** Returns address of project owner. */
-export default function useOwnerOfProject(projectId: BigNumberish | undefined) {
+export default function useOwnerOfProject(projectId?: BigNumberish) {
```

Comment:
I was really curious why `undefined` was explicitly set. I should have just asked rather than just making this change. That makes sense. Additionally, this change shouldn't have been in here, to begin with.